services/server/config/config.go

```diff
@@ -203,6 +205,11 @@ func LoadConfig(path string, out *Config) error {
 			continue
 		}
 
+		if _, err := os.Stat(path); errors.Is(err, os.ErrNotExist) {
```

Comment:
Where is this change used? This looks like separate a behavior change.

Reply:
@dmcgowan My bad for not explaining it better. Yes this is a separate behaviour change. 

Today if the config.toml file does not exist, 'containerd config dump' returns an error. But technically, if the config.toml file does not exist, it means that the configuration is default. So instead of throwing an error, this returns 'containerd config default' instead.

Reply:
That change should occur in ctr, not in the server

Reply:
As mentioned, there are two commands which this pull request deals with. One of them is ctr, but containerd config dump is not a ctr command, hence the change is required here.

Reply:
It looks like the containerd command line already tries to address the case when the config.toml file does not exist [here](https://github.com/containerd/containerd/blob/f4a905109b7a98b37157d11f560b20300ca87ca0/cmd/containerd/command/config.go#L111).
But, it seems that the `LoadConfig` function never returns an `os.ErrNotExist` error, so the check doesn't work as expected. That is why the change is happening in the server. Feel free to correct me if I interpreted this wrong @abhayanoop.

Reply:
@Jenkins-J  I think you have put it absolutely right. Thanks!